### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29367]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Ryan Thurlwell (OCTO Design Lead)

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
On the compare, there is a confirmation alert when there is no acutal alert.

### Link, screenshot or steps to recreate
https://staging.va.gov/gi-bill-comparison-tool-sandbox/compare/?facilities=14952435%2C10040835%2C21853035

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Remove alert styling for "This school doesn’t have any caution flags"

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by watersemma** [comment]
Updates have been made based on feedback. Sketch file can be found here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/school-comparison-tool/redesign/design/sketch-files/gibct-redesign-comparison-postmvp.sketch

**Comment by af-manish** [comment]
@rtwell  Gentle nudge here?

**Comment by rtwell** [comment]
lgtm!

**Comment by ERRonat0609** [comment]
Adding screen examples for Dev/QA: on the comparison page, when there is no caution alert for a school, right now there is formatting added that should be removed. 

Currently: 
![image](https://user-images.githubusercontent.com/85580424/141007806-40cfd49f-50d2-4aea-8d2f-fe5ac22b7542.png)

Special alert formatting should be reserved for alerts only. 
![image](https://user-images.githubusercontent.com/85580424/141007462-6f5df9a1-eb1d-4eb2-b509-405199789bb4.png)

**Comment by nbarthelemy** [comment]
Updated PR with minor classname change: https://github.com/department-of-veterans-affairs/vets-website/pull/19740

**Comment by hallihocine** [comment]
Ticket has been validated in staging: The "Cautionary information" section contains ONLY styling alerts for institutions that have caution flag. see below: 
<img width="1332" alt="Screen Shot 2021-12-21 at 2 59 29 PM" src="https://user-images.githubusercontent.com/86266024/146993259-a3152e14-5de4-4179-817e-cbc0074f822e.png">

**Comment by hallihocine** [comment]
- Test Run in Test Rail: 
https://dsvavsp.testrail.io/index.php?/runs/view/2744&group_by=cases:section_id&group_order=asc&group_id=3251
- Test Script/Steps:
[Design components or patterns don't align with Design System guidelines. (04.07.3).xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7758053/Design.components.or.patterns.don.t.align.with.Design.System.guidelines.04.07.3.xlsx)

**Comment by mrblanco1** [comment]
EDU UAT occurred and passed on 12/30/2021

**Comment by mrblanco1** [comment]
Hi @rtwell and @shiragoodman - Good afternoon! After having reviewed this ticket with EDU/Joe Preisser and his team, "they thought this fix/solution looks unfinished. There was also a concern that the non-caution text size was too disproportionate to the caution text, which could be a problem for someone with a visual impairment. If we cannot use the same formatting as the caution flag, can we at least increase the font so that it’s more proportionate to the caution font?"

Can you please help us with recommendations/suggestion regarding the above feedback?  Thanks!

**Comment by rtwell** [comment]
feel free to increase the text size.

**Comment by mrblanco1** [comment]
Thank you @rtwell - I will let EDU know and plan the change accordingly.
